calendar.php

```diff
@@ -899,7 +899,7 @@
 		{
 			$privatecheck = '';
 		}
-		$start_date = explode("-", gmdate("j-n-Y-g:i A", $event['starttime']+$event['timezone']*3600));
+		$start_date = explode("-", gmdate("j-n-Y", $event['starttime']+$event['timezone']*3600));
```

Comment:
Any reason time was completely removed instead of using the `timeformat` setting? Same applies to the line below.